src/App.tsx

```
@@ -1,10 +1,106 @@
+function handle() {
+  // ❌ Generic name
+  return "handled";
+}
+
+function doSomething() {
+  // ❌ Generic name
+  return "done";
+}
+
+// ❌ Intentional type mismatch to show violation
+if ((A as any) === (B as any)) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)